spec/requests/api/v1/admin/trends/links/links_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Links' do
+  let(:role)    { UserRole.find_by(name: 'Admin') }
+  let(:user)    { Fabricate(:user, role: role) }
+  let(:scopes)  { 'admin:read admin:write' }
+  let(:token)   { Fabricate(:accessible_access_token, resource_owner_id: user.id, scopes: scopes) }
+  let(:headers) { { 'Authorization' => "Bearer #{token.token}" } }
+
+  describe 'GET /api/v1/admin/trends/links' do
+    subject do
+      get '/api/v1/admin/trends/links', headers: headers
+    end
+
+    it 'returns http success' do
+      subject
+
+      expect(response).to have_http_status(200)
+    end
+  end
```

Comment:
I'd like to improve the coverage for this but I wasn't able to do the setup. How could I make a trending link?

Reply:
I *think* a `PreviewCardTrend` object simply needs to be created. This is different for other types of trending items, though.